addons/common/stringtable.xml

```diff
@@ -564,14 +564,17 @@
         <Key ID="STR_ACE_Common_toggleHandheldDevice">
             <English>Toggle Handheld Device</English>
             <Spanish>Seleccionar dispositivo de mano</Spanish>
+            <German>Handheld-Gerät aufrufen</German>
         </Key>
         <Key ID="STR_ACE_Common_closeHandheldDevice">
             <English>Close Handheld Device</English>
             <Spanish>Cerrar dispositivo de mano</Spanish>
+            <German>Handheld-Gerät schließen</German>
```

Comment:
Ich bin mir nicht sicher, ob das Wort "Gerät" in diesem Fall eine Redundanz darstellt...